tests/unit/commands/internal/test_dispatch.py

```
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from menv.commands.internal import dispatch
+
+
+class TestDispatch:
+    def test_raises_file_not_found_when_binary_missing(self, tmp_path: Path) -> None:
```

Ruff (0.15.2)

[warning] Unused method argument: `tmp_path`

(ARG002)